BrowserKit/Sources/WebEngine/Security/ContentBlockingOptions.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import WebKit
+
+public struct ContentBlockingSettings: OptionSet {
```

Comment:
nit; filename should be the same as the struct it contains

Reply:
Thanks for catching, I had renamed this and forgot to update the filename. Fixed now